packages/plugin-angular/src/index.ts

```diff
@@ -1,6 +1,17 @@
 import { ErrorHandler, Injectable } from '@angular/core'
 import Bugsnag, { Client, Plugin } from '@bugsnag/js'
 
+// That's the `global.Zone` exposed when the `zone.js` package is used.
+declare const Zone: any
```

Comment:
```suggestion
// angular uses zones to watch for changes in asynchronous tasks so it can
// update the UI in response
// Bugsnag uses a lot of asynchronous tasks when notifying, which trigger change
// detection multiple times. This causes a potential performance problem, so we
// need to run `notify` outside of the current zone if zones are being used
// see https://angular.io/guide/zone
declare const Zone: any
```